Upstream-Status: Backport
Signed-off-by: Ross Burton <ross.burton@intel.com>

From e51d67549b516b2dac6c71d92c8499f6e67125ad Mon Sep 17 00:00:00 2001
From: Thorsten Kukuk <kukuk@thkukuk.de>
Date: Tue, 14 Nov 2017 10:43:53 -0500
Subject: [PATCH] Fix typo in src/libtirpc.map

Which prevents that key_secretkey_is_set will be exported.

Signed-off-by: Thorsten Kukuk <kukuk@suse.de>
Signed-off-by: Steve Dickson <steved@redhat.com>
---
 src/libtirpc.map | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libtirpc.map b/src/libtirpc.map
index f385de5..21d6065 100644
--- a/src/libtirpc.map
+++ b/src/libtirpc.map
@@ -298,7 +298,7 @@ TIRPC_0.3.2 {
     key_gendes;
     key_get_conv;
     key_setsecret;
-    key_secret_is_set;
+    key_secretkey_is_set;
     key_setnet;
     netname2host;
     netname2user;
-- 
1.8.3.1
